app/views/ladder/LadderView.js

```diff
@@ -477,6 +482,27 @@ module.exports = (LadderView = (function () {
 
     isAILeagueArena () { return _.find(utils.arenas, { slug: this.levelID }) }
 
+    updateSpectateList () {
+      if (!this.level?.isType('ladder')) return
+      this.leaderboardRankings = this.ladderTab?.rankings || []
+      this.renderSelectors('.spectate-players')
+    }
+
+    onClickNewSpectateButton () {
+      const player1Index = parseInt(this.$el.find('#spectate-player-1').val().split(':')[0]) - 1
+      const player2Index = parseInt(this.$el.find('#spectate-player-2').val().split(':')[0]) - 1
+      const player1 = this.leaderboardRankings[player1Index]
+      const player2 = this.leaderboardRankings[player2Index]
+      const humanSession = player1 != null ? player1[player1.length - 1] : undefined
+      const ogreSession = player2 != null ? player2[player2.length - 1] : undefined
+      let url = `/play/spectate/${this.level.get('slug')}?`
+      if (humanSession && ogreSession) { url += `session-one=${humanSession}&session-two=${ogreSession}` }
+      if (this.league) { url += '&league=' + this.league.id }
+      if (key.command) { url += '&autoplay=false' }
```

Comment:
what is this key.command used here and in window.open?

Reply:
it's an old logic. i think it's for when you `command + click` button in mac 😂